packages/@expo/prebuild-config/src/plugins/icons/withAndroidIcons.ts

```diff
@@ -177,8 +183,15 @@ export async function configureAdaptiveIconAsync(
   projectRoot: string,
   foregroundImage: string,
   backgroundImage: string | null,
+  monochromeImage: string | null,
   isAdaptive: boolean
 ) {
+  if (monochromeImage)
```

Comment:
Wrap with curly brackets

Reply:
fixed as of [5570270](https://github.com/expo/expo/pull/20074/commits/55702708369cfc4d377a3b29b49c943c9e357a6e) (build commit coming after)